### PR TITLE
GEOMESA-2065 Add json4s to HBase libjars

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <ahulbert@ccri.com>

### CODE REVIEW

**Comment by amcdonaldccri** [comment]
looks good

File: geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list [hunk 1]
```diff
@@ -22,8 +25,11 @@ mango
 metrics
 minlog
 netty
+paranamer
 parboiled
+Saxon-HE
```
Comment (jnh5y) [inline geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list]: What happens if the user has *not* installed the Saxon-HE jar?

Comment (jahhulbert-ccri) [reply geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list]: won't get pulled. just skipped.

Comment (jahhulbert-ccri) [reply geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list]: https://github.com/locationtech/geomesa/blob/master/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/classpath/ClassPathUtils.scala#L33

**Comment by jahhulbert-ccri** [comment]
ok this should be ready enough now.

**Pull request merged**